mobile/src/main/java/org/openhab/habdroid/ui/CloudNotificationListFragment.kt

```diff
@@ -84,6 +85,9 @@ class CloudNotificationListFragment : Fragment(), View.OnClickListener, SwipeRef
         recyclerView.layoutManager = layoutManager
         recyclerView.addItemDecoration(DividerItemDecoration(view.context))
         recyclerView.adapter = adapter
+
+        // Clear notifications from statusbar
+        CloudMessagingHelper.clearAllNotifications(view.context)
```

Comment:
I'd move this call to `onResume()` as the notification list is loaded there as well. You can also drop the comment, the function name is self-explanatory.

Reply:
Then it would send the FCM `hideNotification()` mesage every time onResume() is called. Is there any way to check if there are any active notifications, prior to Android N ?

Reply:
@mueller-ma my method needs non-null context, is it safe to pass `context!!` if I move the method call to `onResume()` ?

Reply:
Yes, it is (even though I don't like using the `!!` operator, there's no other choice there).